apps/site/assets/ts/stop/components/BasicTime.tsx

```diff
@@ -20,8 +21,16 @@ const BasicTime = ({
       ? formatRelativeTime(time, targetDate)
       : formatToBostonTime(time, "h:mm aa");
 
+  // displayType === "relative" might produce an absolute-formatted time if it's
+  // more than an hour away. parse the result to find the truth.
+  const parsedTime = parse(formattedTime, "h:mm aa", new Date()).getTime();
+  const isAbsolute = Number.isNaN(parsedTime) ? "relative" : "absolute";
```

Comment:
question: what is an `absolute-formatted time`?

Reply:
@i0sea 😅 Ryan had this same question on an earlier PR, maybe I need a better name for it.
It's the format that's not relative... absolute = 9:55AM, relative = 20 minutes

Reply:
maybe `standard format` and `minutes format` although thats a bit wordy 💭